client/scss/components/_slider.scss

```diff
@@ -55,25 +57,37 @@ $slider-control-margin: 18px;
   }
 }
 
-.slider__slides-container { // these width and left calculations are related to the slider appearing in the article template
+.slider__slides-container {
   margin: 0;
   position: relative;
   width: calc(100vw - #{map-get($container-padding, 'small')});
 
   @include respond-to('medium') {
-    width: calc(100vw - #{map-get($container-padding, 'medium')} - 12.5%);
-    margin-left: 12.5%;
+    width: calc(100vw - #{map-get($container-padding, 'medium')});
+
+    .slider__slider--in-article & {
+      width: calc(100vw - #{map-get($container-padding, 'medium')} - 12.5%);
+      margin-left: 12.5%;
+    }
   }
 
   @include respond-to('large') {
     width: calc(100vw - #{map-get($container-padding, 'large')});
-    margin-left: 0;
+
+    .slider__slider--in-article & {
+      margin-left: 0;
+    }
```

Comment:
Having parent specificity always . scares me a little, and isn't very BEM (which should be as flat as possible).
Could we not just add a modifier `slider__slides-container--no-margin` instead?